src/main/java/hudson/plugins/jira/JiraMailAddressResolver.java

```diff
@@ -20,8 +20,18 @@
 public class JiraMailAddressResolver extends MailAddressResolver {
     private static final Logger LOGGER = Logger.getLogger(JiraMailAddressResolver.class.getName());
 
+    /**
+     * Boolean to disable the Jira mail address resolver.
+     *
+     * To disable set the System property "-Dhudson.plugins.jira.JiraMailAddressResolver.DISABLE=true"
+     */
+    public static final boolean DISABLE = Boolean.getBoolean(JiraMailAddressResolver.class.getName() + ".DISABLE");
```

Comment:
> then I can use a groovy script to set the value

Exactly. You'll be able to test whether this has the desired effect while troubleshooting without having to reboot the production instance.